_posts/2021-02-21-goodbye-scrum.md

```diff
@@ -0,0 +1,79 @@
+
+Kanban is more than just an improved visualization system, though. Kanban isn't Kanban without WIP limits. WIP limits mean we will intentionally limit starting new tasks until (enough) current tasks are completed. This swings the pendulum from optimizing for utilization (where everyone has "something to do" and isn't sitting idle or looking for work) to a focus on completing work. This clearly addresses the stalled-project problem we're trying to solve by focusing efforts on completing work, thereby contributing to forward progress and momentum. Another virtuous cycle!
+
+WIP limits have the added benefit of codifying a social contract around when we pair and mob. By playing with the WIP limits, we can tweak how frequently we mob. If, for example, we can only have three tasks in progress at any given time, then by definition some of us will be mobbing (or all of us will be pairing). It changes the conversation from an open-ended ad hoc plea of "I'm doing X, feel free to pair/mob with me!" to a more specific "I cannot pick up a new ticket, so I'm going to mob with you."
+
+## Our Own Flavor of Kanban
+
+What this means for us is that we need to better visualize our work and institute WIP limits. Better visualization will take the form of additional columns in Jira. To start with, I'm proposing the following: IDEA → SPECIFY → TODO → DEV → VERIFY → DONE. These states are meant to get us started with something useful, and I fully expect (indeed: insist) to tweak and adjust as we learn more. To see what this looks like, check out [this test] project in Jira. I'll be migrating us to this Classic project, as it has some important features that are missing in Next-Gen projects (namely, indication of how long a task is in a column).
+
+The IDEA will contain items that are SWAGged, i.e. they'll contain just enough information that we were able to come up with a SWAG. This helps us have ROI conversations that inform the priority of these items.
+
+From IDEA, items move into SPECIFY. These are essentially on-deck items. This is where (and when) on-demand planning happens. When a SPECIFY item is picked up, the owner's responsibility is to break it down into max 2-3 day tickets in the TODO column. The result is a more detailed plan. But the TODO column does have a WIP limit. If a SPECIFY task's decomposition is too big to fit into the TODO column, then break the SPECIFY task apart in-place in order to adhere to our limits by specifying smaller chunks of better-understood work.
+
+The items in TODO are from where new work is picked up. When you're ready to pull a new task, grab the topmost ticket from TODO that you've got the skills to do. (This is where we can inject escalated bugs, etc.) Or, better yet, pick up the top task — and if you're not sure how to do it, pair with somebody that does!
+
+When picking up an item from TODO, you'll move it into DEV. If you cannot because there's already too many tasks in that column, you pair/mob!
```

Comment:
`If you cannot because there's already too many tasks in that column, you pair/mob!` -> `If you cannot because there are already too many tasks in that column, you pair/mob!`